themes/bootstrap3/templates/RecordDriver/DefaultRecord/list-entry.phtml

```diff
@@ -239,9 +239,10 @@
   </div>
   <?php
     $escId = $this->escapeJs($id);
+    $md5Id = md5($id);
```

Comment:
Why not move this higher in the template so you can use the same value in line 229 as well instead of calculating the md5 twice?

Reply:
Additionally, you can remove the `$escId` variable, which is no longer used after this change.

Reply:
You are right, the md5 sum can be calculated once, but we still need the `$escId` for the `delete` parameter of the post request. Shall I make a new pull request?

Reply:
You can make the changes in this branch and the current PR will update; no need to create a second one. Thanks!